File: basemodule/src/main/java/zjm/cst/dhu/basemodule/presenter/BasePresenter.java
package zjm.cst.dhu.basemodule.presenter;

import zjm.cst.dhu.basemodule.view.BaseView;

/**
 * Created by zjm on 2017/2/24.
 */

public interface BasePresenter<T extends BaseView> {
    void attachView(T BaseView);

    void detachView();
}

File: basemodule/src/main/java/zjm/cst/dhu/basemodule/presenter/BasePresenter.kt
package zjm.cst.dhu.basemodule.presenter

import zjm.cst.dhu.basemodule.view.BaseView

/**
 * Created by zjm on 2017/2/24.
 */

interface BasePresenter<T : BaseView> {
    fun attachView(BaseView: T)

    fun detachView()
}
